contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -1,4 +1,57 @@
+
+== Library Repositories
+
+The Boost libraries are modular, and each library has its own separate repository.  This makes it easier to work with the entire collection of Boost libraries.
+
+The official Boost repositories are hosted on GitHub at https://github.com/boostorg/[Boost.org]. Each library has its own repository under this organization. For example, https://github.com/boostorg/json for boost:json[], or https://github.com/boostorg/date_time/ for boost:date_time[].
+
+As with any Git-based project, you can clone the repositories, make changes, commit those changes to your local repository, and push your changes back to the server. If you want to contribute changes to the official Boost libraries, you can make a pull request, which the maintainers of the libraries can then review and potentially merge into the official codebase. Be aware though that Boost has a well-defined workflow for contributors, including guidelines for how to submit changes, how changes are reviewed, and so forth. Refer to xref:getting-involved.adoc[].
+
+== The Super-project Repository
+
+The Boost https://github.com/boostorg/boost[*Super-project*] repository includes all the libraries as submodules.
+
+In addition to the libraries, you'll find many meta-repositories under `boostorg` that handle other tasks, for example:
+
+* https://github.com/boostorg/website2022[Website generation]
+* https://github.com/boostorg/cmake[Cmake support]
+* https://github.com/boostorg/regression[Regression testing]
+* https://github.com/boostorg/release-tools[Release tools]
+
+There is also a https://github.com/boostorg/boost/tree/master/tools[tools] directory in the Super-project, which contains various tools used for tasks such as building and testing Boost libraries. 
+
+The *master* branch of the Super-project is usually the development version of Boost, where new features are added and bugs are fixed. When there is a public release of Boost, it is built from the Super-project repo.
+
+== Bug and Issue Reporting
+
+Apart from the code repositories, the `boostorg` organization also hosts other resources related to Boost development, such as issue trackers for each library, and discussion forums (in the form of GitHub Discussions).
+
+To report an issue, locate the *New Issue* button in the repo for the library. For example, https://github.com/boostorg/json/issues for the boost:json[] library.
+
+When reporting a bug or issue, you might be offered a range of templates to choose from to guide your filing. If there is no template to follow, cut-and-paste the following into the text field for the issue, and replace the text in italics with appropriate details.
+
+### Version of <Library/Boost>
+
+_If you can find the version of the library itself (in a readme file, other documentation, or a header file), then put that version number here. Otherwise, if you have difficulty finding a version for the library itself, add the Boost version number, such as `1.82.0`._
```

Comment:
If you gear this towards a user, I'd tell them to just include the Boost version number, since I'd say the author knows the internal version (if any) for a given Boost version.

Reply:
@anarthal Good point on the issue templates. I have the basics of how to create one, but it would be great to point out a good example - so please go ahead and add one to the MySQL, and I'll link to it. Thanks